src/AcmTile/AcmTile.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import '@patternfly/react-core/dist/styles/base.css'
+import React, { useState } from 'react'
+import { Meta } from '@storybook/react'
+import { AcmTile } from './AcmTile'
+
+const meta: Meta = {
+    title: 'Tile',
+    component: AcmTile,
+    argTypes: {
+        title: { type: 'string' },
+        message: { type: 'string' },
+    },
+}
+export default meta
+
+export const Tile = () => {
+    const [selected, toggleSelected] = useState<boolean>(false)
+
+    return (
+        <AcmTile
+            isSelected={selected}
+            title={''}
+            onClick={() => toggleSelected(!selected)}
+            relatedResourceData={{ count: 10, kind: 'Related pod' }}
+        />
```

Comment:
You may want to render multiple states of the component in the storybook.  In this particular case it will be helpful to render a tile with a count > 1,000.  You should also check that it renders correctly with a long kind string.

I think of storybook as a "visual testcase".  There's storybook plugins to enhance that usage, we could change the inputs dynamically or even report the stories as automates tests.  We can explore those later, but for now adding more stories is an easy way to test the component.

Reply:
Yea I was debating adding a couple more tiles to be shown in the storybook. I didnt because we are eventually going to have a container component and was thinking I would add multiple there.

Reply:
That also works, it adds another layer in the code, but it easier to see all in the same container at once.

I'm ok either way as long as we have a way to visualize the different states.